src/orders/components/OrderFulfillPage/OrderFulfillPage.tsx

```diff
@@ -184,6 +184,35 @@ const OrderFulfillPage: React.FC<OrderFulfillPageProps> = props => {
       items: formsetData
     });
 
+  const shouldEnableSave = () => {
```

Comment:
- short circuit and exit early
- clear variable names
- logic kept near logic, not inside Component tree

Love it